trade-finance-manager-api/src/v1/controllers/deal.party-db.js

```diff
@@ -1,5 +1,33 @@
 const api = require('../api');
 
+/**
+ * Gets company information from Party URN
+ * @param {Express.Request} req
+ * @param {Express.Response} res
+ * @returns {Object} Company information
+ */
+const getCompany = async (req, res) => {
+  try {
+    const { urn } = req.params;
+
+    if (!urn) {
+      return res.status(404).send(false);
+    }
+
+    const company = await api.getCompanyInfo(urn);
+
+    if (!company) {
+      console.error('Void company response received');
+      return res.status(404).send(false);
+    }
+
+    return res.status(200).send(company);
+  } catch (e) {
+    console.error('Unable to get company: ', { e });
```

Comment:
Not a blocker - maybe worth adding "Party URN" to the console logs for future debugging?